src/components/Text/styles.scss

```diff
@@ -0,0 +1,74 @@
+@import "../../styles/variables.scss";
+
+@mixin HeadingText {
+  color: $color_white;
+  font-family: "Omnes";
+  font-weight: 700;
+  line-height: 125%;
+}
+
+@mixin BodyText {
+  color: $color_white;
+  font-family: "Operator Mono";
+  font-weight: 350;
+  line-height: 125%;
+  margin-top: $spacing_xx_small;
+  margin-bottom: $spacing_xx_small;
+}
+
+.H1 {
```

Comment:
nit but could we lowercase these class selectors?

Reply:
also what do you think about selecting the elements, i.e. via `.App h1 { styles }` as opposed to needing to designate a className for every element? I think that could help us when it comes to usage by just needing to use the right tag instead of passing text, classname, style, and type to a text component via props

Reply:
Yeahh down to lowercase, was just following the style guide in Figma so that it matches